### ISSUE TITLE
App.asar over 1gb

### ISSUE THREAD

**Issue by martijnmichel** [SimulatedGREG/electron-vue#600]
Im creating a cardgame orion with this template, it works like a charm. But i do have a problem.

When i first build the app it was already 200mb, 2nd time it was 500mb and now the 3rd time ive build it the app.asar takes up 1.4gb of space!

Ive searched internet but i really dont know whats going on.. the files and images in the rendered folder take up 600kb.

Can anyone help me out here?
Thanks

**Comment by vieruu** [comment]
I think this happens because you added to your app.asar the dist/build folder

**Comment by martijnmichel** [comment]
Where would i have done that? I dont remember changing anything in config files (except installing some plugins in --save-dev mode)

**Comment by vieruu** [comment]
extract your app.asar using the asar node module and see what is taking 1GB

**Comment by martijnmichel** [comment]
This is in the build.config.js
module.exports = {
  arch: 'x64',
  asar: true,
  dir: path.join(__dirname, '../'),
  icon: path.join(__dirname, '../build/icons/icon'),
  ignore: /(^\/(src|test|\.[a-z]+|README|yarn|static|dist\/web))|\.gitkeep/,
  out: path.join(__dirname, '../build'),
  overwrite: true,
  platform: process.env.BUILD_TARGET || 'all'
}

**Comment by vieruu** [comment]
have you tried extracting the archive yet ?

**Comment by martijnmichel** [comment]
Yes! I had created directories with builds for backup.. they were included in the new file. 
Thanks that solves it.
